test/ui-test/testSuites/suite_settings/tst_userIdentity/test.feature

```diff
@@ -2,36 +2,65 @@ Feature: User Identity
 
     As a user I want to set my identity, that is: display name, bio and social links.
 
+    The feature start sequence follows the global one (setup on global `bdd_hooks`): No additional steps
 
-    The feature start sequence is the following (setup on its own `bdd_hooks`):
-    ** given A first time user lands on the status desktop and generates new key
-    ** when user signs up with username "tester123" and password "TesTEr16843/!@00"
-    ** and the user lands on the signed in app
+    Background: Sign up and open settings section
+        Given A first time user lands on the status desktop and generates new key
+        And the user signs up with username "tester123" and password "TesTEr16843/!@00"
+        And the user lands on the signed in app
+        Given the user opens app settings screen
 
-	# TODO: Scenario broken due to new user profile design. It will be addressed in task #8281
-	@mayfail
-    Scenario: User sets display name, bio and social links
-        When the user opens app settings screen
+
+
+    Scenario Outline: The user sets display name, bio and social links
+        Given the user opens app settings screen
```

Comment:
We already do this one in the Background

Reply:
Oh yeah.. I'll remove it from the background. Thanks!